package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "codius-manifest",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
Can you revert this?  We should be doing the version changes are part of a tagging process.